frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/contextmenu/SlideInNewBranchBelowSelectedBranchAction.java

```diff
@@ -0,0 +1,16 @@
+package com.virtuslab.gitmachete.frontend.actions.contextmenu;
+
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import io.vavr.control.Option;
+
+import com.virtuslab.gitmachete.frontend.actions.base.BaseSlideInNewBranchBelowAction;
+import com.virtuslab.gitmachete.frontend.actions.expectedkeys.IExpectsKeySelectedBranchName;
+
+public class SlideInNewBranchBelowSelectedBranchAction extends BaseSlideInNewBranchBelowAction
```

Comment:
Yeah, as mentioned elsewhere... it should ideally be `slide in new or existing branch`